Classes/Controllers/PBGitCommitController.h

```diff
@@ -9,43 +9,16 @@
 #import <Cocoa/Cocoa.h>
 #import "PBViewController.h"
 
-@class PBGitIndexController;
-@class PBIconAndTextCell;
-@class PBWebChangesController;
 @class PBGitIndex;
-@class PBNiceSplitView;
-@class PBCommitMessageView;
 
-@interface PBGitCommitController : PBViewController {
-	// This might have to transfer over to the PBGitRepository
-	// object sometime
-	PBGitIndex *index;
-
-	IBOutlet PBCommitMessageView *commitMessageView;
-
-	BOOL stashKeepIndex;
-
-	IBOutlet NSArrayController *unstagedFilesController;
-	IBOutlet NSArrayController *cachedFilesController;
-	IBOutlet NSArrayController *trackedFilesController;
-
-	IBOutlet NSTabView *controlsTabView;
-	IBOutlet NSButton *commitButton;
-	IBOutlet NSButton *stashButton;
-
-	IBOutlet PBGitIndexController *indexController;
-	IBOutlet PBWebChangesController *webController;
-	IBOutlet PBNiceSplitView *commitSplitView;
-}
-
-@property(readonly) PBGitIndex *index;
-@property(assign) BOOL stashKeepIndex;
+@interface PBGitCommitController : PBViewController
 
 - (IBAction) refresh:(id) sender;
```

Comment:
Just wondering: What qualifies IBActions to appear in the Interface?

My POV would be that all IBActions should be part of the interface as their role is to be called from the outside.